os_migrate/plugins/module_utils/user.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import openstack
+
+from ansible_collections.os_migrate.os_migrate.plugins.module_utils \
+    import const, resource
+
+
+class User(resource.Resource):
+    resource_type = const.RES_TYPE_USER
+    sdk_class = openstack.identity.v3.user.User
+
+    info_from_sdk = [
+        'domain_id',
+        'is_enabled',
+    ]
+
+    params_from_sdk = [
+        'name',
+        'password',
```

Comment:
This is surprising - can we really migrate the password? I think this probably doesn't work and should be in _info, no?

And on the contrary, there are additional things that we probably do want to migrate - is_enabled, description, maybe default_project_name (via reference id/name mechanisms) too?

Here are the docs: https://docs.openstack.org/openstacksdk/latest/user/resources/identity/v3/user.html#openstack.identity.v3.user.User 

Domain_id is probably correctly in _info. Trying to save a domain_name reference would probably break it for Keystone v2, and i think the users/projects will anyway be created in the domain which is provided in admin credentials, if any.

Reply:
Re password - it probably doesn't make sense to have it in _info either, i doubt the API would return the acutal password or even the stored hash, so we can probably drop it altogether?

Reply:
I think we should leave the password and iterate over, looks like a bit useless to migrate 1000s of users that need to restart their password? Everything else is fixed.